dev-docs/docs/introduction/contributing.mdx

```diff
@@ -1,3 +1,11 @@
 # Contributing
 
 Pull requests are welcome. For major changes, please [open an issue](https://github.com/excalidraw/excalidraw/issues/new) first to discuss what you would like to change.
+
+See [CONTRIBUTING.md](https://github.com/excalidraw/excalidraw/blob/master/CONTRIBUTING.md) for more.
```

Comment:
we should move contributing guidelines over here, but before we do that I'm at least linking over there